mgclient_cpp/include/mgclient.hpp

```diff
@@ -150,8 +151,11 @@ inline std::unique_ptr<Client> Client::Connect(const Client::Params &params) {
   }
   mg_session_params_set_host(mg_params, params.host.c_str());
   mg_session_params_set_port(mg_params, params.port);
+  mg_session_params_set_scheme(mg_params, params.scheme.c_str());
```

Comment:
let's be consistent here: either we always set parameter even if empty, either check if empty field